library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -293,87 +288,78 @@ SAMLBinding.doc=Bindings for the OASIS Security Assertion Markup Language (SAML)
   Composite
 SAMLBinding.uri=https://www.oasis-open.org/committees/download.php/56780/sstc-saml-bindings-errata-2.0-wd-06-diff.pdf
 
-SAMLBindings_3_1_2_1=servers MUST authenticate to clients using a X.509 v3 certificate
+SAMLBindings_3_1_2_1_a=servers MUST authenticate to clients using a X.509 v3 certificate
```

Comment:
✏️ Uppercase `S` or `[S]` or add the beginning of the sentence

Reply:
Added entire sentence.